docs/mine/lotus/miner-lifecycle.md

```diff
@@ -58,11 +74,59 @@ Deadline Challenge:   638 (in 1h10m30s)
 Deadline FaultCutoff: 588 (in 45m30s)
 ```
 
-Once you have verified that your miner is safe to shut down, you can run `lotus-miner stop`. If you are using systemd, run `systemctl stop lotus-miner`.
+The `proving info` examples above show information for the current proving window and deadlines. If you wish to see any upcoming deadlines you can use:
+
+```bash
+$ lotus-miner proving deadlines
+```
 
-You can restart the miner as soon as you wish. Any sealing operations that were ongoing will be reset to the last checkpoint and continue.
+Every row corresponds to a deadline (a period of 30 minutes covering 24 hours). The current one is marked. This is sometimes useful to find a time of day in which the miner does not have to submit any proofs to the chain.
+
+### Checking and temporally disabling deals
+
+Before stopping the miner, you may check the state of your deals to make sure the miner is not receiving data or retrieving data for a client:
+
+```bash
+lotus-miner storage-deals list
+lotus-miner retrieval-deals list
+lotus-miner data-transfers list
+```
+
+To prevent new deals from coming in while you wait to finish work for the current deadline etc, you may want to disable storage and retrieval deals so that the miner does not find itself in the middle of a new one when shutdown:
```

Comment:
```suggestion
To prevent new deals from coming in while you wait to finish work for the current deadline, you can disable storage and retrieval deals. This ensures that the miner does not find itself in the middle of a new deal when shutting down:
```